tests/php/Forms/UsedOnTableTest.php

```diff
@@ -27,11 +27,11 @@ public function testExcludes()
 
         $expected = json_encode([
             "usage"=> [[
-                "id" => 0,
+                "id" => 1,
                 "title" => "My Page",
                 "type" => "Page",
-                "state" => "Draft",
-                "link" => sprintf('%s://localhost/admin/pages/edit/show/1', $protocol)
+                "link" => sprintf('%s://localhost/admin/pages/edit/show/1', $protocol),
```

Comment:
Can we avoid explicitly referencing localhost here? Otherwise this test will only pass when your test environment is on localhost.

Reply:
I don't know.   'localhost' was part of an existing test that was works fine as is.  I don't think spending time here is worthwhile.

Reply:
This fix doesn't take a lot of time to apply and it's going to save people like me who run their test suite under a different BASE_URL a lot of aggravation.

```suggestion
                "link" => Director::absoluteURL('admin/pages/edit/show/1'),
```

Reply:
Updated